src/ares_iq/typing.py

```diff
@@ -1,28 +1,38 @@
+"""Protocols and types for the ares_iq package."""
+
 from typing import Protocol
 from .iq_data import IQData
 import os
 import pathlib
 
+QuantizedData = None
+"""Type alias for the quantized data."""
+
+PathLike = str | pathlib.Path | os.PathLike | bytes
+"""Type alias for paths."""
+
+RealNumber = int | float
+"""Type alias for numbers that are considered "real" in math terms."""
+
 
 class SoftwareDefinedRadio(Protocol):
+    """Protocol for SoftwareDefinedRadio classes."""
+
     def capture_iq(self, center: float, bw: float, file_size: float, verbose: bool, extra_verbose: bool) -> None:
-        """
-        Capture IQ data from the SDR.
-        :param center: The center frequency in Hz
-        :param bw: The bandwidth in Hz
-        :param file_size: The maximum amount of IQ data to collect in GB
-        :param verbose: Show progress bar.
-        :param extra_verbose: Extra verbose output. Shows the progress bar and logging messages.
-        :return: The captured IQ data and the
+        """Capture IQ data from the SDR.
+
+        Args:
+            center: The center frequency in Hz.
+            bw: The bandwidth in Hz.
+            file_size: The maximum amount of IQ data to collect in GB.
```

Comment:
Declare file sizes in int:bytes instead of float:GB to get ahead of potential floating point issues (someone will try to declare some exact file_size and get in trouble due to rounding)